python/ml4ir/base/config/default_model_config.yaml

```diff
@@ -24,6 +24,9 @@ optimizer:
 lr_schedule:
   key: constant
   learning_rate: 0.01
+calibration:
+  key: temperature_scaling
```

Comment:
should we calk it `method` instead of `key`? it is fair to point that the same is happening to the `optimizer` and `lr_schedule` sections though.

Reply:
not sure about this one and how exactly it differs for `calibration`. @lastmansleeping  could u comment on that ?

Reply:
Should be fine for now. We might want to rename these things like @balikasg mentions later.